framework/src/multiapps/MultiApp.C

```diff
@@ -112,6 +112,11 @@ validParams<MultiApp>()
       false,
       "If true this will cause the output from the MultiApp to be 'moved' by its position vector");
 
+  params.addParam<Real>(
+      "start_time",
+      0,
+      "The start time of the application in terms of the global time. When this parameter is "
+      "non-zero. The global time will be ahead by the offset specified here.");
```

Comment:
The master and subapp starting times may not be zero, so `start_time` might be misleading. Since this parameter is used in `app->setGlobalTimeOffset(start_time);`, why not just name it as `global_time_offset`. I would say in docstring *this parameter can be used to change the starting time of the subapps*. In the docstring, the second period should be a comma.